docs/admin/integrations/platformx.md

```diff
@@ -8,21 +8,88 @@ their experience.
 
 ![PlatformX Events in Coder](../../images/integrations/platformx-screenshot.png)
 
+Use these steps to forward Coder notifications to PlatformX using Docker and a
+lightweight web server.
+
+For more information about this integration and how it works, refer to the
+[coder-platformx-notifications](https://github.com/coder/coder-platformx-notifications)
+repository.
+
 ## Requirements
 
 You'll need:
 
-- A running Coder v2.19+ deployment
+- Coder v2.19+
 - A PlatformX subscription from [DX](https://getdx.com/)
-- A platform to host the integration (choose any of these):
+- A platform to host the integration, such as:
   - AWS Lambda
+  - Google Cloud Run
   - Heroku
   - Kubernetes
   - Or any other platform that can run Python web applications
 
-## Getting Started
+## Run coder-platformx-events-middleware with Docker
 
-Refer to the instructions in the
-[coder-platformx-notifications](https://github.com/coder/coder-platformx-notifications)
-repository to deploy a lightweight web server to forward Coder Notifications to
-PlatformX.
+Coder sends [notifications](../monitoring/notifications/index.md) via webhooks
+to coder-platformx-events-middleware (CPEM), which processes and reformats the payload
+into a structure compatible with [PlatformX by DX](https://help.getdx.com/en/articles/7880779-getting-started).
+CPEM transforms the payload and forwards it to PlatformX for further processing and analysis.
+
+CPEM is optimized for serverless environments such as Google Cloud Run and AWS Lambda.
+
+Clone the repository and use Docker to build and run CPEM:
```

Comment:
Since CPEM is a bit of a workaround/intermediary step (see #16264), let's avoid this acronym. also I imagine users will want to preview the repo and modify the source code so we should def link to the repo.

By linking to the repo, we'll need to think about what info we include in the README versus docs. e.g. "where is the source of truth"